aspnetcore/blazor/debug.md

```diff
@@ -1,123 +1,170 @@
 ---
-title: Debug ASP.NET Core Blazor
+title: Debug ASP.NET Core Blazor WebAssembly
 author: guardrex
 description: Learn how to debug Blazor apps.
 monikerRange: '>= aspnetcore-3.1'
 ms.author: riande
 ms.custom: mvc
-ms.date: 01/16/2020
+ms.date: 03/24/2020
 no-loc: [Blazor, SignalR]
 uid: blazor/debug
 ---
-# Debug ASP.NET Core Blazor
+# Debug ASP.NET Core Blazor WebAssembly
 
 [Daniel Roth](https://github.com/danroth27)
 
 [!INCLUDE[](~/includes/blazorwasm-preview-notice.md)]
 
-*Early* support exists for debugging Blazor WebAssembly using the browser dev tools in Chromium-based browsers (Chrome/Edge). Work is in progress to:
+Blazor WebAssembly apps can be debugged using the browser dev tools in Chromium-based browsers (Edge/Chrome).  Alternatively you can debug your app using Visual Studio or Visual Studio Code.
 
-* Fully enable debugging in Visual Studio.
-* Enable debugging in Visual Studio Code.
-
-Debugger capabilities are limited. Available scenarios include:
+Available scenarios include:
 
 * Set and remove breakpoints.
-* Single-step (`F10`) through the code or resume (`F8`) code execution.
-* In the *Locals* display, observe the values of any local variables of type `int`, `string`, and `bool`.
+* Run the app with debugging support in Visual Studio and Visual Studio Code (<kbd>F5</kbd> support).
+* Single-step (<kbd>F10</kbd>) through the code.
+* Resume code execution with <kbd>F8</kbd> in a browser or <kbd>F5</kbd> in Visual Studio or Visual Studio Code.
+* In the *Locals* display, observe the values of local variables.
 * See the call stack, including call chains that go from JavaScript into .NET and from .NET to JavaScript.
 
 You *can't*:
 
-* Observe the values of any locals that aren't an `int`, `string`, or `bool`.
-* Observe the values of any class properties or fields.
-* Hover over variables to see their values.
-* Evaluate expressions in the console.
-* Step across async calls.
-* Perform most other ordinary debugging scenarios.
+* Inspect arrays.
+* Hover to inspect members.
+* Step debug into or out of managed code.
+* Have full support for inspecting value types.
+* Break on unhandled exceptions.
+* Hit breakpoints during app startup.
+* Debug an app with a service worker.
 
-Development of further debugging scenarios is an on-going focus of the engineering team.
+We expect to continue to improve the debugging experience in future releases.
```

Comment:
My own perception of "future" is a bit vague and it leaves me with an impression that this may not happen.
What about slightly modified version below (please ignore this, if you find your phrasing better):

```suggestion
We will continue to improve the debugging experience in upcoming releases.
```